src/goose/defaults.clj

```diff
@@ -11,12 +11,16 @@
 (def ^:no-doc in-progress-queue-prefix "goose/in-progress-jobs:")
 (def ^:no-doc process-prefix "goose/processes:")
 (def ^:no-doc heartbeat-prefix "goose/heartbeat:")
+(def ^:no-doc batch-prefix "goose/batch:")
 
 (def default-queue "default")
 (def schedule-queue "scheduled-jobs")
 (def dead-queue "dead-jobs")
 (def cron-queue "cron-schedules")
 (def cron-entries "cron-entries")
+(def enqueued-job-set "enqueued")
+(def successful-job-set "successful")
```

Comment:
suggestion: can you add a set for retrying jobs?

Reply:
maybe that can be added in your story @alishamohanty ?

Reply:
Yeah, I intentionally left that out. We can add all retry related changes in a single commit.